components/kyma-environment-broker/internal/broker/plans.go

```diff
@@ -422,3 +430,12 @@ func filter(items *[]interface{}, included map[string]interface{}) interface{} {
 
 	return output
 }
+
+func isEURestrictedAccess(platformRegion string) bool {
```

Comment:
this function will be used for another task (HAP) and in my opinion could be moved to the internal package (or other place, let's discuss)

Reply:
Done, committed and pushed.